commands_to_load/Mod.py

```diff
@@ -79,5 +79,390 @@ def __init__(self, bot):
         eObj = em(title='A Bellow From the Underworld says...', colour=0xff0000, author=ctx.author.display_name, avatar=ctx.author.avatar_url, description=msg, footer='Moderator Warning')
         await ctx.send(embed=eObj)
 
+    @commands.command(aliases=['propm'])
+    async def propigatemute(self, ctx):
+        """Ensures all channels have the muted role as part of its permissions.""" 
+        ## Avoid the admin category of channels and rules and announcments channel since nobody can talk in there anyway
+
+        logger.info('[Mod propigatemute()] propigatemute function detected by user ' + str(ctx.message.author))
+        await ctx.message.delete()
+        logger.info('[Mod propigatemute()] invoking message deleted')
+        
+        if not ctx.message.author in discord.utils.get(ctx.guild.roles, name="Minions").members:
+            logger.info('[Mod propigatemute()] unathorized command attempt detected. Being handled.')
+            await self.rekt(ctx)
+            return
+
+        MUTED_ROLE = discord.utils.get(ctx.guild.roles, name='Muted')
+
+        ignoreChannels = [

```

Comment:
are you sure you will never use these in another command? cause it might be better to assign them to the bot itself rather than just make it an array inside one command

Reply:
moved to settings.py